### DISCUSSION TITLE
Fullscreen (height & width, no padding or margin) modal

### DISCUSSION THREAD

**Discussion by epicmau5time** [mantinedev/mantine#1708] [Feature requests]
How could I have a modal actually take up all of the viewport? No outside padding or anything, similar to bootstrap 5's [fullscreen modal](https://getbootstrap.com/docs/5.0/components/modal/#fullscreen-modal). I've tried styling it myself but nothing seems to work.

**Comment by KaRkY** [comment]
Already supported in v5
https://v5.mantine.dev/changelog/5-0-0/#fullscreen-modal

**Reply by epicmau5time** [reply]
Ohh. The docs and npm all point to v4 by default. Is v5 still in beta or could I use as is by targeting npm i @mantine/core@5.0.0?

**Reply by epicmau5time** [reply]
Nvm. Was on mobile when I clicked the link so didn't see the alpha tag on v5. Thanks for the heads up anyway.
